src/legacy/core_plugins/kibana/public/dashboard/panel/dashboard_panel_container.ts

```diff
@@ -17,26 +17,66 @@
  * under the License.
  */
 
-import { connect } from 'react-redux';
-import PropTypes from 'prop-types';
 import { i18n } from '@kbn/i18n';
-
-import { DashboardPanel } from './dashboard_panel';
-import { DashboardViewMode } from '../dashboard_view_mode';
-
+import { connect } from 'react-redux';
+import { Action } from 'redux-actions';
+import { ThunkDispatch } from 'redux-thunk';
+import {
+  ContainerState,
+  EmbeddableFactory,
+  EmbeddableMetadata,
+  EmbeddableState,
+} from 'ui/embeddable';
+import { CoreKibanaState } from '../../selectors';
 import {
-  deletePanel, embeddableError, embeddableIsInitialized, embeddableIsInitializing, embeddableStateChanged,
+  deletePanel,
+  embeddableError,
+  EmbeddableErrorAction,
+  embeddableIsInitialized,
+  embeddableIsInitializing,
+  embeddableStateChanged,
 } from '../actions';
-
+import { DashboardViewMode } from '../dashboard_view_mode';
 import {
+  getContainerState,
   getEmbeddable,
+  getEmbeddableError,
+  getEmbeddableInitialized,
   getFullScreenMode,
+  getPanel,
+  getPanelType,
   getViewMode,
-  getEmbeddableError,
-  getPanelType, getContainerState, getPanel, getEmbeddableInitialized,
+  PanelId,
+  PanelState,
 } from '../selectors';
+import { DashboardPanel } from './dashboard_panel';
+
+export interface DashboardPanelContainerOwnProps {
+  panelId: string | PanelId;
```

Comment:
`PanelId` is already typed as a string?

Reply:
Passes as string but not as PanelId...

Reply:
This passed jest tests locally when run today so going to commit the changes since it makes sense to keep this to PanelId versus both string or PanelId.